ijson/backends/yajl2_c.py

```diff
@@ -0,0 +1,26 @@
+#
+# Contributed by Rodrigo Tobar <rtobar@icrar.org>
+#
+# ICRAR - International Centre for Radio Astronomy Research
+# (c) UWA - The University of Western Australia, 2016
+# Copyright by UWA (in the framework of the ICRAR)
+# All rights reserved
+#
+'''
+Wrapper for _yajl2 C extension module
+'''
+import decimal
+
+from ijson import common
+from ._yajl2 import basic_parse as _basic_parse  # @UnresolvedImport
+from ._yajl2 import parse as _parse  # @UnresolvedImport
+from ._yajl2 import items as _items  # @UnresolvedImport
```

Comment:
Importing the C module itself instead of its individual parts sounds fine, I will change that.

Regarding the name, what is the motivation for having pyyajl2 instead of _yajl2? In the python library python wrappers for C-written seem to be usually named likewise (e.g. ssl imports _ssl, socket imports _socket, struct imports _struct). Following that I named the C module _yajl2.c, without the final "c" in the name as it is already present in the extension. I think also that the name pyyajl2 somehow would imply that the extension is written in pure python, which in turn is confusing again since there is already a python backend.

Let me know if you want to reconsider the naming; otherwise I will go ahead with the names you suggest.